modules/Components/src/client/ACASFormFields.coffee

```diff
@@ -1034,7 +1034,7 @@ class ACASFormLSBooleanFieldController extends ACASFormAbstractFieldController
 
 	renderModelContent: =>
 		# If value is anything other than true (i.e. null), then default to unchecked
-		if @getModel().get('value') is "true"
+		if @getModel().get('value').toLowerCase() is "true"
```

Comment:
I just commented about this on [#394](https://github.com/mcneilco/acas-roo-server/pull/394) about how we are storing upper case true values in the migration.

Also, did you check that this also work in the case where the value is null?

Reply:
Looked at it, not sure if I fully understand...whether it is stored as "TRUE"/"FALSE" or "true"/"false", this should be OK right? 

Will look into what happens if it is null.